pkg/autoscaler/aggregation/bucketing_test.go

```diff
@@ -81,7 +80,7 @@ func TestTimedFloat64BucketsSimple(t *testing.T) {
 			// New implementation test.
 			buckets := NewTimedFloat64Buckets(2*time.Minute, tt.granularity)
 			for _, stat := range tt.stats {
-				buckets.Record(stat.time, stat.name, stat.value)
```

Comment:
Should we remove `pod` too? Also `name` from struct `args`.

Reply:
We cannot remove it from stat, since we're using it in the collector.
So here I can use an empty string, of course, but this is more expressive I think.